content/tutorials/manage-the-rippled-server/installation/build-run-rippled-ubuntu.md

```diff
@@ -9,16 +9,9 @@ Use these instructions to build a `rippled` executable from source version 1.1.0
 For information about building `rippled` for other platforms, see [Builds](https://github.com/ripple/rippled/tree/develop/Builds) in the `rippled` GitHub repository.
 
 
-## System Requirements
-
-**_To build `rippled`:_**
-
-You need a **minimum** of 8GB of RAM.
```

Comment:
The minimum requirements now include 8GB of RAM just to run rippled, so this separation is no longer relevant